app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] Kindly make sure you add the `new` action in your controller allowing a user to add a new comment as required for the project